prep-javascript-objects/main.js

```diff
@@ -0,0 +1,18 @@
+var person = {
+  firstName: 'Patricia',
+  lastName: 'Manalang',
+  hobby: 'Watching TikToks'
+};
+console.log(person);
+var firstName = 'Patricia ';
+var lastName = 'Manalang ';
+var fullName = firstName + lastName;
```

Comment:
Hi Patricia, for this exercise you can get the first name and last name of the person using dot notation.

```js
var fullName = person.firstName + ' ' + person.lastName;
```